ledger/src/blockstore.rs

```diff
@@ -103,7 +103,7 @@ lazy_static! {
 
 pub const MAX_REPLAY_WAKE_UP_SIGNALS: usize = 1;
 pub const MAX_COMPLETED_SLOTS_IN_CHANNEL: usize = 100_000;
-pub const MAX_TURBINE_PROPAGATION_IN_MS: u64 = 100;
+pub const MAX_TURBINE_PROPAGATION_IN_MS: u64 = 200;
```

Comment:
Maybe change this to `std::time::Duration` and remove `_IN_MS` from the name; and avoid `Duration::from_millis` here:
https://github.com/solana-labs/solana/blob/f24e30e4f/core/src/repair_generic_traversal.rs#L259

`duration.as_millis() as u64` is `const` so it should be fine to define the next `const`.